app/src/main/java/com/example/dine_and_donate/HomeFragments/MapFragment.java

```diff
@@ -420,4 +448,67 @@ private void slideDownMenu() {
         slideView.startAnimation(animate);
     }
 
+    private void setUpCarousel(final DataSnapshot snapshot) {
```

Comment:
i don't totally follow the point of this carousel (probably cause i don't have a visual), can you attach a screenshot?